src/app/users/home/home.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+import { AuthService } from '../../services/auth.service';
+import { Router } from '@angular/router';
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css'],
+})
+export class HomeComponent {
+  isSignedIn = false;
+  constructor(
+    private AuthService: AuthService,
+    private router: Router
+  ) {}
+  logout() {
+    this.isSignedIn = false;
+    this.AuthService.signOut();
```

Comment:
call AuthService first then only make isSignedIn= false, else due to network issue you will generate bug.

if possible make isSignedIn false base on reposes